example/src/app/simple/__snapshots__/simple.component.spec.ts.snap

```diff
@@ -4,7 +4,6 @@ exports[`SimpleComponent snapshots 1`] = `
 <app-simple>
   <p>
      simple works!
-  
```

Comment:
Is it okay that I removed that empty lanes from existing snapshots? it is because of added filter function in serializer.